### PR TITLE
fix(auth): Don't use import in js files

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- This fails to start auth

## This pull request

- Don't do this, I checked other places where we might possibly have this issue and it doesn't appear anywhere

## Checklist

- [ ] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Review by chenba — APPROVED** [review]

**Comment by chenba** [comment]
kinda expecting a linter to complain about that.

**Comment by pdehaan** [comment]
Interesting. I mean, I guess why this wouldn't work, but... not sure how to grep/scan for other potential instances.

It looks like we have almost 3K *.js files w/ `import` statements, although that obviously isn't bad since we might have `"type": "module"` in the package.json or maybe are using esbuild...

```sh
git grep -En "^import " packages/**/*.js | wc -l # 2977

git grep -En "^import " packages/**/*.js | head -15
packages/fxa-admin-panel/src/setupTests.js:5:import '@testing-library/jest-dom/extend-expect';
packages/fxa-auth-server/lib/routes/totp.js:16:import { Container } from 'typedi';
packages/fxa-auth-server/lib/routes/totp.js:17:import { AccountEventsManager } from '../account-events';
packages/fxa-auth-server/test/local/routes/subscriptions/mozilla.js:4:import { MozillaSubscriptionTypes } from 'fxa-shared/subscriptions/types';
packages/fxa-auth-server/test/local/routes/subscriptions/mozilla.js:6:import { ERRNO } from '../../../../lib/error';
packages/fxa-auth-server/test/local/routes/subscriptions/paypal-notifications.js:35:import { RefundType } from '../../../../lib/payments/paypal/client';
packages/fxa-auth-server/test/local/routes/subscriptions/paypal-notifications.js:36:import { SUBSCRIPTIONS_RESOURCE } from '../../../../lib/payments/stripe';
packages/fxa-content-server/app/scripts/app.js:4:import 'core-js';
packages/fxa-content-server/app/scripts/app.js:5:import {
packages/fxa-content-server/app/scripts/app.js:9:import ConfigLoader from './lib/config-loader';
packages/fxa-content-server/app/scripts/app.js:10:import AppStart from './lib/app-start';
packages/fxa-content-server/app/scripts/head/boot.js:5:import StartupStyles from './startup-styles';
packages/fxa-content-server/app/scripts/head/startup-styles.js:11:import Environment from '../lib/environment';
packages/fxa-content-server/app/scripts/lib/app-start.js:18:import _ from 'underscore';
packages/fxa-content-server/app/scripts/lib/app-start.js:19:import ExperimentGroupingRules from './experiments/grouping-rules/index';
```

**Comment by pdehaan** [comment]
Actually, looks like a lot of the *.js `import` statements are in `/test/`, `/tests/` or fxa-content-server `/scripts/` directories, so some negative greps shrinks that down to a reasonable 20 results:

TLDR:

| PACKAGE | COUNT |
|:----|----:|
| `fxa-admin-panel` | 1
| `fxa-auth-server` | 2
| `fxa-payments-server` | 8
| `fxa-shared/speed-trap` | 9

```sh
git grep -En "^import " packages/**/*.js | grep -Ev "/tests?/" | grep -v "/scripts/" | wc -l #  20

git grep -En "^import " packages/**/*.js | grep -Ev "/tests?/" | grep -v "/scripts/"
packages/fxa-admin-panel/src/setupTests.js:5:import '@testing-library/jest-dom/extend-expect';
packages/fxa-auth-server/lib/routes/totp.js:16:import { Container } from 'typedi';
packages/fxa-auth-server/lib/routes/totp.js:17:import { AccountEventsManager } from '../account-events';
packages/fxa-payments-server/src/lib/logger.test.js:5:import chai from 'chai';
packages/fxa-payments-server/src/lib/logger.test.js:6:import Logger from './logger';
packages/fxa-payments-server/src/lib/logger.test.js:7:import sinon from 'sinon';
packages/fxa-payments-server/src/lib/sentry.js:5:import Logger from './logger';
packages/fxa-payments-server/src/lib/sentry.js:6:import * as Sentry from '@sentry/browser';
packages/fxa-payments-server/src/lib/sentry.js:8:import {
packages/fxa-payments-server/src/lib/sentry.test.js:5:import chai from 'chai';
packages/fxa-payments-server/src/lib/sentry.test.js:6:import sentryMetrics from './sentry';
packages/fxa-shared/speed-trap/guid.js:5:import { v4 as uuidv4 } from 'uuid';
packages/fxa-shared/speed-trap/index.js:9:import { default as SpeedTrap } from './speed-trap';
packages/fxa-shared/speed-trap/navigation-timing.js:5:import {
packages/fxa-shared/speed-trap/performance-factory.js:5:import { NAVIGATION_TIMING_FIELDS } from './timing-fields';
packages/fxa-shared/speed-trap/speed-trap.js:4:import guid from './guid';
packages/fxa-shared/speed-trap/speed-trap.js:5:import NavigationTiming from './navigation-timing';
packages/fxa-shared/speed-trap/speed-trap.js:6:import Timers from './timers';
packages/fxa-shared/speed-trap/speed-trap.js:7:import Events from './events';
packages/fxa-shared/speed-trap/speed-trap.js:8:import { getPerformanceApi } from './performance-factory';
```
